src/bilder/images/edxapp/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -409,17 +409,24 @@ REGISTRATION_EXTRA_FIELDS:
     terms_of_service: hidden
     year_of_birth: optional
 RETIRED_EMAIL_DOMAIN: retired.invalid
-RETIRED_EMAIL_PREFIX: retired__user_
-RETIRED_USERNAME_PREFIX: retired__user_
-RETIRED_USER_SALTS:
-- OVERRIDE ME WITH A RANDOM VALUE
-- ROTATE SALTS BY APPENDING NEW VALUES
+RETIRED_EMAIL_PREFIX: retired__user__
+RETIRED_USERNAME_PREFIX: retired__user__
 RETIREMENT_SERVICE_WORKER_USERNAME: retirement_worker
 RETIREMENT_STATES:
-- PENDING
-- ERRORED
-- ABORTED
-- COMPLETE
+    - PENDING
+    - LOCKING_ACCOUNT
+    - LOCKING_COMPLETE
+    - RETIRING_FORUMS
+    - FORUMS_COMPLETE
+    - RETIRING_EMAIL_LISTS
+    - EMAIL_LISTS_COMPLETE
+    - RETIRING_ENROLLMENTS
+    - ENROLLMENTS_COMPLETE
+    - RETIRING_LMS
+    - LMS_COMPLETE
+    - ERRORED
+    - ABORTED
+    - COMPLETE
```

Comment:
Where are all those states defined?

Reply:
This is mostly a copy of https://github.com/openedx/edx-platform/blob/master/lms/envs/common.py#L4827-L4854